packages/palette/src/elements/Carousel/utils.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Sums all values, building from the previous
+ * @param xs list of values to compound
+ */
+export const compound = (xs: number[]): number[] => {
+  return xs.reduce((p, n) => p.concat((+p.slice(-1) || 0) + n), [])
+}
+
+/**
+ * Build an array of offsets
+ * @param viewport window width
+ * @param values array of cell widths
+ */
+export const chunk = (viewport: number, values: number[]) => {
```

Comment:
I see - these are more lodash-like utils. Could we move them up to the top level utils folder?